libraries/standard/http/cbmc/proofs/demo/Makefile

```diff
@@ -0,0 +1,82 @@
+METHOD=IotMqtt_Disconnect
+ENTRY=harness
+
+################################################################
+# Proof assumptions
+
+# Bound the number of subscriptions in a list (BOUND = MAX-1)
+SUBSCRIPTION_COUNT_MAX=2
+DEF += -DSUBSCRIPTION_COUNT_MAX=$(SUBSCRIPTION_COUNT_MAX)
+
+# allow mqtt mallocs to fail for coverage
+DEF += -include mqtt_state.h
+DEF += -DIotMqtt_MallocMessage=malloc_can_fail
+DEF += -DIotMqtt_MallocOperation=malloc_can_fail
+
+################################################################
+
+SRCDIR ?= $(abspath $(PROOFDIR)/../../..)
+
+INC += \
```

Comment:
Is this different for each proof? Or can it go in the base makefile?

Reply:
In proof hierarchies that have proofs at differing depths in the proof hierarchy, in this case (and in FreeRTOS), each proof is going to have to define SRCDIR rising up differing amounts to the source root.